packages/yoastseo/grunt/config/shell.js

```diff
@@ -18,7 +18,7 @@ module.exports = function( grunt ) {
 			" --force-po" +
 			" --from-code=UTF-8" +
 			" --add-comments=\"translators: \"" +
-			" --add-comments=\"Translators: \"" +
```

Comment:
I would maybe leave this, if someone makes the mistake to capitalize 'translators', at least with this line of code the comment would end up on translate.wordpress.org. What do you think?

Also, if you do want to remove it, there is no need to replace it with the uncapitalized version, as that already exists on the line above 😄

Reply:
Okay, I will leave it then, thank you @agnieszkaszuba !